### DISCUSSION TITLE
Taxa with structural zero

### DISCUSSION THREAD

**Discussion by Bio2406839** [FrederickHuangLin/ANCOMBC#255] [Q&A]
Hi! I have a question regarding this sentence in the tutorial: "A taxon is considered to have structural zeros in some (>=1) groups if it is completely (or nearly completely) missing in these groups."

I was wondering what is the abundance threshold in at least one group to assign a taxa with structural zero.

Thanks!

**Comment by simone-anza** [comment]
I think it's 0 if  struc_zero = T, but can be also very close to 0 if neg_lb = T
